src/System.Data.SQLite/SqliteStatementPreparer.cs

```diff
@@ -60,6 +60,10 @@ public SqliteStatementHandle Get(int index, CancellationToken cancellationToken)
 							Thread.Sleep(20);
 							break;
 
+						case SQLiteErrorCode.Interrupt:
+							cancellationToken.ThrowIfCancellationRequested();
```

Comment:
If this is correct, it seems like it would also be appropriate on line 58, but perhaps it's safer to make as few changes as possible.